README.md

```diff
@@ -8,35 +8,36 @@ software program.
 ## Job components and plan
 
 To run BLAST, we need three things: 
-- the BLAST program (specifically the `blastx` binary)
-- a reference database (this is usually a larger file)
-- the file we want to query against the database
+1. the BLAST program (specifically the `blastx` binary)
+2. a reference database (this is usually a larger file)
+3. the file we want to query against the database
 
-The database and the input file will each get special treatment. The database we're using 
+The database and the input file will each get special treatment. The database we are using 
 is large enough that we will want to use OSG Connect's `stashcache` capability (more information 
 about that [here][stashcache]). The input 
-file is large enough that a) it's near the upper limit of what's practical to transfer and 
+file is large enough that a) it is near the upper limit of what is practical to transfer, 
 b) it would take hours to complete a single `blastx`
-analysis for it and the resulting output file would be huge. Because the BLAST process is 
-run over the input file line by line, this input file can be split into smaller pieces and 
-the queries can be run as separate jobs. This is what we're going to do below. 
+analysis for it, and c) the resulting output file would be huge. 
+
+Because the BLAST process is 
+run over the input file line by line, it is scientifically valid to split up the input query file, analyze the pieces, and then put the results back together at the end! By splitting the input query file into smaller pieces, each of the queries can be run as seporate jobs. On the other hand, BLAST databases should not be split, because the blast output includes a score value for each sequence that is calculated relative to the entire length of the database.
```

Comment:
```suggestion
run over the input file line by line, it is scientifically valid to split up the input query file, analyze the pieces, and then put the results back together at the end! By splitting the input query file into smaller pieces, each of the queries can be run as separate jobs. On the other hand, BLAST databases should not be split, because the blast output includes a score value for each sequence that is calculated relative to the entire length of the database.
```